mandatory/2-function-creation.js

```diff
@@ -25,7 +39,20 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(array) {
```

Comment:
For this function, it's possible to call multiple methods when returning. So you could have el.trim().toLowerCase().replace(/\//g, '')